examples/ex39.cpp

```diff
@@ -0,0 +1,288 @@
+//                                MFEM Example 39
+//
+// Compile with: make ex39
+//
+// Sample runs:  ex39
+//               ex39 -ess "Southern Boundary"
+//               ex39 -src Base
+//
+// Device sample runs:
+//               ex39 -fa -d cuda
```

Comment:
Uncovered by the nightly tests: this sample run is not valid -- the option `-fa` is not supported at the moment.